fastf1/api.py

```diff
@@ -41,38 +43,37 @@
 
 from fastf1.utils import recursive_dict_get, to_timedelta, to_datetime
 
-
 base_url = 'https://livetiming.formula1.com'
 
 headers = {
-  'Host': 'livetiming.formula1.com',
-  'Connection': 'close',
-  'TE': 'identity',
-  'User-Agent': 'BestHTTP',
-  'Accept-Encoding': 'gzip, identity',
+    'Host': 'livetiming.formula1.com',
```

Comment:
You seem to have auto-formatted the whole file. While not wrong, I'd prefer to not have all these changes in this PR if possible.

Reply:
Looks like an IDEA bug to be honest. Didn't show up as changed in my editor but they where changed. Reverted.